src/cycax/cycad/features.py

```diff
@@ -9,7 +9,9 @@ class Holes(Location):
         x : The location of x along the x axis.
         y : The location of y along the y axis.
         z : The location of z along the z axis.
-        side : The side of the odject that this location refers to. This will be used to specify from which side a feature should be inserted into another object. This will be one of TOP, BOTTOM, LEFT, RIGHT, FRONT, BACK.
+        side : The side of the odject that this location refers to.
```

Comment:
Is "side :" and "side:" the same, please check the generated docs. If its the same then its OK, we use "side:" in most places.